src/Immunization-1-0/Cert/PHR-Client/imm-retrieve-1-2.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-model href="http://hl7.org/fhir/STU3/testscript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <TestScript xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript" nts:scenario="client">
-    <id value="imm-retrieve-5-1"/>
```

Comment:
Dit bestand staat in de PHR-Client-map, wat doet vermoeden dat het een retrieve scenario is. Waarom wordt dat nu "serve-"?